onnxruntime/core/providers/dnnl/math/gemm.cc

```diff
@@ -92,7 +92,7 @@ template <typename T>
 using ConstEigenMatrixMapRowMajor = Eigen::Map<const Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>>;
 
 template <>
-Status Gemm<float>::Compute(Provider_OpKernelContext* ctx) const {
+Status Gemm<float>::Compute(Provider_OpKernelContext* ctx, const Provider_OpKernel_Base&) const {
```

Comment:
>const Provider_OpKernel_Base&) [](start = 59, length = 30)

Can this be hidden somewhere in a base class and accessible thru some kind of Get()?

Reply:
The problem was that the real base class on the internal side passes one value in to the constructor, makes a copy, then uses that in the later calls. I don't really like how it works currently either. I'll keep thinking about it...